<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filter</class>
 <widget class="QDialog" name="filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Database:</string>
     </property>
     <widget class="QRadioButton" name="radio_frommainwindow">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>From MainWindow</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_fromfile">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>From File:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="le_import">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_fimport">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output File Path:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="le_export"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_fexport">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Filter Option:</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="radio_fdrug">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>64</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Drug</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_flead">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Lead</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_ffragment">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Fragment</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_fcustom">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Custom</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="check_funcitonalgroup">
     <property name="text">
      <string>Filter Functional Groups</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="autoExpandDelay">
      <number>-1</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pb_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
